db/schema.rb

```diff
@@ -107,6 +107,28 @@
     t.datetime "shortlisted_at"
     t.datetime "unsuccessful_at"
     t.datetime "withdrawn_at"
+    t.string "first_name", default: "", null: false
+    t.string "last_name", default: "", null: false
+    t.string "previous_names", default: "", null: false
+    t.string "street_address", default: "", null: false
+    t.string "city", default: "", null: false
+    t.string "postcode", default: "", null: false
+    t.string "phone_number", default: "", null: false
+    t.string "teacher_reference_number", default: "", null: false
+    t.string "national_insurance_number", default: "", null: false
+    t.string "qualified_teacher_status", default: "", null: false
```

Comment:
I'd like to call this general area of the schema 'professional_status' rather than 'qualified_teacher_status', because that's harder to confuse with 'qualifications'.

Reply:
Ignore this!